src/middlewared/middlewared/plugins/etc.py

```diff
@@ -144,6 +144,10 @@ class EtcService(Service):
              'local_path': 'local/proftpd.conf'},
             {'type': 'py', 'path': 'local/proftpd'},
         ],
+        'kdump': [
+            {'type': 'mako', 'path': 'default/kdump-tools',
+             'local_path': 'default/kdump-tools.mako', 'platform': 'Linux'},
```

Comment:
Can't we just strip `.mako` from template file name and thus avoid repeating ourselves?